src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDropinXmlDocument.java

```diff
@@ -68,4 +139,47 @@ public void createFeature(String name) {
         child.appendChild(text);
         featureManager.appendChild(child);
     }
+
+    public Element findFeatureManager() {
+        Element serverElement = findServerElement();
+        if (serverElement == null) {
+            return null;
+        }
+        Node fmNode = serverElement.getElementsByTagName("featureManager").item(0); // assume just one
```

Comment:
Can `getElementsByTagName("featureManager")` return null if the `featureManager` element is not found?  If so, should check for a null pointer before calling `.item(0);`

Reply:
`getElementsByTagName("featureManager")` will always return a NodeList which can be empty. `item()` will return a null if the index is out of bounds and not throw an exception like IndexOutOfBounds so this is ok. `fmNode` could be null in which case the `instanceof` fails.